zenoh/src/net/runtime/adminspace.rs

```diff
@@ -72,6 +72,10 @@ impl AdminSpace {
 
         let mut handlers: HashMap<_, Handler> = HashMap::new();
         handlers.insert(root_key.clone(), Arc::new(router_data));
+        handlers.insert(
+            format!("@/router/{zid_str}/metrics").try_into().unwrap(),
```

Comment:
Should we call it `@/router/{zid_str}/stats` instead of `@/router/{zid_str}/metrics` for consistency with the rest?

Reply:
`metrics` is the industry standard.